test.py

```diff
@@ -6474,6 +6474,14 @@ def test_generate_data_matrix():
     )
 
 
+@skip_if_not_modified
+def test_rseqc_inner_distance():
+    run(
+        "bio/rseqc/inner_distance",
+        ["snakemake", "--cores", "1", "--use-conda", "-F"],
```

Comment:
Maybe be a bit more specific:
```suggestion
        ["snakemake", "--cores", "1", "--use-conda", "-F", "a.pdf"],
```